File: Backend/sModels/CompanyOwnerAddress.cs
using System;
using System.Collections.Generic;

namespace WebAPIBackend.sModels;

public partial class CompanyOwnerAddress
{
    public int Id { get; set; }

    public int? AddressTypeId { get; set; }

    public int? ProvinceId { get; set; }

    public int? DistrictId { get; set; }

    public int? CompanyOwnerId { get; set; }

    public string? Village { get; set; }

    public DateTime? CreatedAt { get; set; }

    public string? CreatedBy { get; set; }

    public bool? Status { get; set; }

    public virtual AddressType? AddressType { get; set; }

    public virtual CompanyOwner? CompanyOwner { get; set; }

    public virtual Location? District { get; set; }

    public virtual Location? Province { get; set; }
}

File: Frontend/src/app/models/companyOwnerAddress.ts
export interface companyOwnerAddress{
    id:number;
    addressTypeId:number;
    provinceId:number;
    districtId:number;
    village:string;
    companyOwnerId:number;
}

export interface companyOwnerAddressData{
    id:number;
    addressTypeId:number;
    provinceId:number;
    districtId:number;
    village:string;
    companyOwnerId:number;
    addressType:string;
    province:string;
    district:string;
}
